RNApp/android/app/build.gradle

```diff
@@ -83,8 +83,8 @@ def enableSeparateBuildPerCPUArchitecture = false
 def enableProguardInReleaseBuilds = false
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.0"
+    compileSdkVersion 23
```

Comment:
ahh we shouldn't have committed this. not a big deal just future reference, this might not be compatible with the sdk versions on others' dev machines